tc-tests-utils.sh

```diff
@@ -47,7 +47,7 @@ assert_correct_ldc93s1()
   assert_correct_inference "$1" "she had your dark suit in greasy wash water all year"
 }
 
-download_material()
+download_native_client_files()
```

Comment:
I am stealing part of that because I need to perform something similar in https://github.com/mozilla/DeepSpeech/pull/810